samples/Simple/main.cpp

```diff
@@ -309,6 +379,49 @@ int main(int /*argc*/, char* /*argv*/[])
         dotnetRuntimeService->freeString(result);
     };
 
+    float timeStep = 1.0f / 60.0f;
+    int32 velocityIterations = 6;
+    int32 positionIterations = 2;
+
+    auto entityId = NovelRT::Atom::getNextEntityId();
+    auto t = boxRect->transform();
+    auto tt = NovelRT::Physics::Box2d::TestTransform{t.position, t.rotation};
+
+    scheduler.GetComponentCache()
+        .GetComponentBuffer<NovelRT::Physics::Box2d::TestTransform>()
+        .PushComponentUpdateInstruction(0, entityId, tt);
+
+    NovelRT::Physics::Box2d::Box2dSystem::AddBody(&world, &groundBodyDef)->CreateFixture(&groundBox, 0);
+    NovelRT::Physics::Box2d::Box2dSystem::AddBodyComponent(&scheduler, &world, entityId, &bodyDef, &boxDef);
+
+    // system.AddBody(&groundBodyDef)->CreateFixture(&groundBox,0);
+    // system.AddEntityBody(&scheduler,entityId,&bodyDef).body->CreateFixture(&boxDef);
+
+    runner.Update += [&](NovelRT::Timing::Timestamp delta) {
+#if false
+        world.Step(delta.getSecondsFloat(), velocityIterations, positionIterations);
+        b2Vec2 position = body->GetPosition();
+        boxRect->transform().position = *reinterpret_cast<NovelRT::Maths::GeoVector2F*>(&position);
+        float angle = body->GetAngle();
+        char message[50];
+        sprintf(message,"%4.2f %4.2f %4.2f\n",position.x, position.y, angle);
+        
+        console.log(message, NovelRT::LogLevel::Debug);
+#else
```

Comment:
What's this for? Testing outside of the ECS setup?

Reply:
Yeah, i forgot to remove it/clean it up. Whoops. Might be good to have an ECS'less example as well that can be toggled, but that could clutter up the main more, and truth be told, it is already quite filled up with a lot of general stuff. Maybe i should look into making a proper physics sample at one point.